frontend/src/features/admin-form/create/builder-and-design/MagicFormBuilder/utils.ts

```diff
@@ -0,0 +1,48 @@
+import * as pdfjs from 'pdfjs-dist/'
+
+pdfjs.GlobalWorkerOptions.workerSrc = new URL(
+  'pdfjs-dist/build/pdf.worker.min.mjs',
```

Comment:
There was an issue on [HVB](https://github.com/opengovsg/signify/pull/283) with non-legacy builds. Something like `'pdfjs-dist/legacy/build/pdf.worker.min.mjs'` might work.

Reply:
changed to the legacy version to potentially support more browsers